modules/cloud-create-instance-steps.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/cloud-create-instance-other.adoc
+// * installing/cloud-create-instance-ocp.adoc
+
+:_content-type: PROCEDURE
+[id="cloud-create-instance-steps_{context}"]
+= Creating an instance in the console
+
+
+toc::[]
+
+[role="_abstract"]
+This topic provides the steps to create a {product-title-managed-short} instance. 
```

Comment:
Does {product-title-managed-short} take an article?

Reply:
I should probably change to "ACS instance" to be consistent with the GUI.